Automattic/wp-calypso
# Downloading media library archive from the export page.

#### Changes proposed in this Pull Request

This PR adds a "Export Media Library" card to the export page:
![image](https://user-images.githubusercontent.com/1842898/46433284-758e3380-c71e-11e8-8786-51304b8ef782.png)


A user can click on the "Download" button in the card to download an archive of the media library.

In summary, it includes:
1. Expand the `siteSettings.exporter` state tree with `mediaExportData`, including actions, selectors and reducers.
1. Data-layer handlers for interacting with `/rest/v1.1/sites/{id}/exports/media`
1. Add a `<ExportMediaCard>` component.

#### Dependencies
This PR is built on https://github.com/Automattic/wp-calypso/pull/27578.

#### Testing instructions
1. Go to http://calypso.localhost:3000/settings/export/
1. The download button should be disabled at the beginning, waiting for the endpoint response.
1. Once receiving the response, the download button should be enabled and you can click on it to download an archive of your media library.
